python-lib/font_exceptions_dict.py

```diff
@@ -1,27 +1,36 @@
 # -*- coding: utf-8 -*-
-"""Module with constants defining the wordcloud font exceptions for languages not supported by Noto Sans Display"""
+"""Module with constants defining the wordcloud font exceptions for languages not supported by the default font.
+- Default font is NotoSansMerged-Regular-1000upem.ttf, which is a merged font based on various NotoSans fonts:
```

Comment:
I would rather keep the module docstring quite short, but have a variable docstring with more details:
```
CONSTANT = "foo"
"""Docstring short

Additional info...
"""
```
See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for reference